<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1" alignment="Qt::AlignTop">
      <widget class="QDateTimeEdit" name="to_datetime_edit">
       <property name="displayFormat">
        <string>yyyy-MM-dd hh:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QDateTimeEdit" name="from_datetime_edit">
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd hh:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignLeft|Qt::AlignBottom">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;To</string>
       </property>
       <property name="buddy">
        <cstring>to_datetime_edit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignBottom">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;From</string>
       </property>
       <property name="buddy">
        <cstring>from_datetime_edit</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
